README.md

```diff
@@ -51,7 +51,7 @@ Include the following jira_source package version in your `packages.yml` file:
 ```yaml
```

Comment:
It won't let me select the line, but just noticed line 49 should say `jira` instead of `jira_source`.

Reply:
good call just updated.